app/models/course.rb

```diff
@@ -1,10 +1,12 @@
 class Course < ActiveRecord::Base
   include Course::LevelsConcern
   include Course::LessonPlanConcern
+  include Componentize
```

Comment:
Do we want to have another Course::ComponentHost instead of mixing the component behaviour into Course? I think that might separate the logic slightly better.